Makefile

```diff
@@ -691,19 +700,19 @@ endif
 
 # create project where the service will be deployed in an OpenShift cluster
 deploy/project:
-	@-oc new-project $(NAMESPACE)
+	@-$(OC) new-project $(NAMESPACE)
 .PHONY: deploy/project
 
 # deploy the postgres database required by the service to an OpenShift cluster
 deploy/db:
-	oc process -f ./templates/db-template.yml | oc apply -f - -n $(NAMESPACE)
+	$(OC) process -f ./templates/db-template.yml | oc apply -f - -n $(NAMESPACE)
 	@time timeout --foreground 3m bash -c "until oc get pods -n $(NAMESPACE) | grep kas-fleet-manager-db | grep -v deploy | grep -q Running; do echo 'database is not ready yet'; sleep 10; done"
```

Comment:
So theres a few other instances similar to the previous comment, I'll just mark them out :)
```diff
-	$(OC) process -f ./templates/db-template.yml | oc apply -f - -n $(NAMESPACE)
-	@time timeout --foreground 3m bash -c "until oc get pods -n $(NAMESPACE) | grep kas-fleet-manager-db | grep -v deploy | grep -q Running; do echo 'database is not ready yet'; sleep 10; done"
+	$(OC) process -f ./templates/db-template.yml | $(OC) apply -f - -n $(NAMESPACE)
+	@time timeout --foreground 3m bash -c "until $(OC) get pods -n $(NAMESPACE) | grep kas-fleet-manager-db | grep -v deploy | grep -q Running; do echo 'database is not ready yet'; sleep 10; done"
```

Reply:
updated